doc/releasenotes/16_0_0_summary.md

```diff
@@ -246,4 +246,13 @@ VSchema Example
 
 #### Flag Deprecations
 
-The flag `lock-shard-timeout` has been deprecated. Please use the newly introduced `lock-timeout` instead. More detail [here](#lock-timeout-introduction).
\ No newline at end of file
+The flag `lock-shard-timeout` has been deprecated. Please use the newly introduced `lock-timeout` instead. More detail [here](#lock-timeout-introduction).
+
+### VTTestServer
+
+#### Improvement
+
+Creating a database takes a lot longer in vttestserver. This can be problematic in the test environment where tests-cases do a lot of `create` and `drop` database.
```

Comment:
```suggestion
Creating a database with vttestserver was taking ~45 seconds. This can be problematic in test environments where testcases do a lot of `create` and `drop` database.
```